library/src/main/java/io/constructor/data/model/search/ResultData.kt

```diff
@@ -0,0 +1,11 @@
+package io.constructor.data.model.search
+
+import com.squareup.moshi.Json
+import io.constructor.data.model.ResultGroup
+
+data class ResultData(val description: String?, val id: String,
+                      @Json(name = "image_url") val imageUrl: String?,
+                      val price: Double?,
+                      val quantity: String?,
+                      val url: String?,
```

Comment:
Re: slack, `price` and `quantity` could exist or not exist per customer.

Reply:
price and qunatity are already nullable, so their their  presence (or not) won't crash anything. Should I add this `slack` property?

Reply:
Nah, `slack` referred to actual Slack.  Like the app Slack.  Is there any way to make ResultData have a mix of typed and untyped properties?